test/append.js

```diff
@@ -121,23 +121,36 @@ tape('append (read back)', function (t) {
 // usual `{ key, value }`. This test adds a bunch of messages very quickly and
 // ensures that the callback contains the correct data.
 tape('append (bulk)', (t) => {
-  const ssb = createSsb();
+  const ssb = createSsb()
 
   // We write 7919 messages, which should be bigger than any cache. It's also a
   // prime number and shouldn't line up perfectly with any batch sizes.
   const messages = 7919
 
-  const assertsPerMessage = 4;
-  t.plan(messages * assertsPerMessage);
+  const assertsPerMessage = 4
+  const plan = messages * assertsPerMessage
+  var pass = 0
+
+  function testEqual (a, b) {
+    if (a !== b) {
+      process.stdout.write('\n')
+      t.equal(a, b)
```

Comment:
this just quietens down this particular test so that if there's an error you see it, but otherwise it doesn't log heaps so you can feasibly scroll past this test in the termina

Reply:
Alternatively:

```suggestion
    if (a !== b) {
      t.fail('messages must be equal')
```